### ISSUE TITLE
Unnecessary counter variable

### ISSUE THREAD

**Issue by inxaneninja** [CactusDuper/pack.png-bot#1]
You can avoid making the counter variable and instead writing str(int(counter)+1) to the file.

**Comment by CactusDuper** [comment]
yeah someone can change that if they want, but i tried to make it as beginner friendly as possible :)
